README.md

```diff
@@ -24,36 +24,29 @@ Worker also expands the set of tables that quartz uses with additional tables to
 5. Run Piezo as specified in [Running](#running).
 
 ###Building
-You must have [sbt](http://www.scala-sbt.org/) version 0.13.0 or higher to build the worker project.
+You must have [sbt](http://www.scala-sbt.org/) version 0.13.0 or higher installed.
 
-To compile the project run
+From the [worker](worker) directory,
 
-`make compile`.
```

Comment:
Not for any Scala/Java project I've seen. aws-sdk-java uses Maven, playframework uses SBT, scala uses Ant, etc.

Mixing build tools is weird.

(Plus, this was a very shallow use of Make. It purpose -- over, say, a shell script -- is to track file dependencies and timestamps, neither of which were being used here.)